.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,26 @@
+---
+name: User Story
+about: Create an issue in the team's standard format
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+#### User Story
+As a issue creator
+
+I want to be able to create issues
+
+So that I can see my issue resolved
+
+#### Acceptance Criteria: 
+- [ ] The issue exists in the list of issues
+- [ ] Issue has all fields properly filled out
+
+#### Tech Notes: 
+- Notes that are technical in nature
+- Not necessarily an acceptance criteria, but a hint for the people that play the card
+
+#### Amigos: 
```

Comment:
Some explanation of what an Amigo is might be helpful as part of this

Reply:
This is for our team to use, and I think we understand each of these sections. What would you propose?

Reply:
+ A list of tagged users that should be involved in reviewing (or exiting) this work

Our team will be creating most of the issues on this repo (for now), however this template will be an option presented on our main open source repo. Anyone adding an issue here will see this as an option so we shouldn't assume that they have an in depth knowledge of terms our team uses.

Additionally, making this clear will help provide documentation for new contributors and developers joining the team.

Reply:
I've changed amigos in the not to be 'users that should be involved in reviewing (or exiting) this work'